src/main/kotlin/org/rust/openapiext/utils.kt

```diff
@@ -40,6 +43,7 @@ fun checkWriteAccessAllowed() {
     }
 }
 
+@Suppress("unused")
```

Comment:
Why?

Reply:
This function is indeed unused, so we probably can just remove it...

But perhaps it should be used somewhere? :) I think all our services are now thread safe, so we don't actually need to enforce  usages of read actions, but it be nice to audit them never the less :)

Reply:
I think it is nice to leave comments every time such annotation is used :)

I thought it might be used via reflection or sth